lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -339,3 +346,28 @@ def test_kerning():
     s = "AVAVAVAVAVAVAVAV€AAVV"
     fig.text(0, .25, s, size=5)
     fig.text(0, .75, s, size=20)
+
+
+def test_glyphs_subset():
+    fpath = str(_get_data_path("fonts/ttf/DejaVuSerif.ttf"))
+    chars = "these should be subsetted!"
+
+    # non-subsetted FT2Font
+    nosubfont = FT2Font(fpath)
+    nosubfont.set_text(chars)
+
+    # subsetted FT2Font
+    subfont = FT2Font(get_glyphs_subset(fpath, chars))
+    subfont.set_text(chars)
+
+    nosubcmap = nosubfont.get_charmap()
+    subcmap = subfont.get_charmap()
+
+    # all unique chars must be available in subsetted font
+    assert set(chars).issuperset(set([chr(key) for key in subcmap.keys()]))
+
+    # subsetted font's charmap should have less (or equal) entries
+    assert len(subcmap) <= len(nosubcmap)
```

Comment:
Shouldn't we require strict inequality here? We specifically want to exclude a lot of glyphs.

Reply:
Testing with font files that are _already_ subsetted with certain characters, the strict equality would fail.

Or for fonts that cannot be subsetted at all.

Reply:
^^ change stuff according to above comment

Reply:
The DejavuSerif.ttf we distribute is not subsetted, so it should get much smaller.